### ISSUE TITLE
qTox fails when calling and then select settings audio / video

### ISSUE THREAD

**Issue by dcapeletti** [qTox/qTox#3603]
##### Brief Description

Trisquel GNU/Linux 7.0 64 bits
qTox version: v1.4.0-387-g34aa905-obs.
Commit hash:  34aa9051e3c001fc30f5d7614f9c4e34354678d8
toxcore:  
Qt:   5.6.1
Hardware:  
…

Reproducible: Couldn't Reproduce
##### Steps to reproduce
1. Calling a customer Antox Android
2. While this calling, go to the Audio / Video settings.
3. QTOX falls.
##### Observed Behavior

QTOX breaks to have an active call and selecting settings audio / video.
##### Expected Behavior

You can select settings audio / video, for example, to change the values in the microphone volume.
##### Additional Info

```
[12:24:20.711 UTC] src/persistence/settings.cpp:134 : Debug: "Loading settings from /home/diegoariel/.config/tox/qtox.ini"
[12:24:20.725 UTC] src/widget/translator.cpp:56 : Debug: Loaded translation "es"
[12:24:20.725 UTC] src/widget/translator.cpp:69 : Debug: System translation not loaded "es"
[12:24:20.725 UTC] src/ipc.cpp:59 : Debug: Our global IPC ID is  14082629075875893316
[12:24:20.725 UTC] src/ipc.cpp:77 : Debug: Attaching to the global shared memory
[12:24:20.726 UTC] src/ipc.cpp:279 : Debug: Previous owner timed out, taking ownership 3973857907430561822 -> 14082629075875893316
[12:24:20.726 UTC] src/main.cpp:204 : Debug: built on:  10:21:47 Aug  5 2016 ( 1470392504 )
[12:24:20.726 UTC] src/main.cpp:205 : Debug: commit:  34aa9051e3c001fc30f5d7614f9c4e34354678d8 

[12:24:20.726 UTC] src/nexus.cpp:84 : Debug: Starting up
[12:24:24.217 UTC] src/persistence/profile.cpp:109 : Debug: Loading tox save  "/home/diegoariel/.config/tox/dcapeletti.tox"
[12:24:24.218 UTC] src/persistence/settings.cpp:404 : Debug: "Saving global settings at /home/diegoariel/.config/tox/qtox.ini"
[12:24:24.330 UTC] src/persistence/settings.cpp:319 : Debug: Loading personal settings from "/home/diegoariel/.config/tox/dcapeletti.ini"
[12:24:24.332 UTC] src/widget/translator.cpp:56 : Debug: Loaded translation "es"
[12:24:24.332 UTC] src/widget/translator.cpp:69 : Debug: System translation not loaded "es"
[12:24:25.274 UTC] src/core/core.cpp:238 : Debug: Loading user profile
[12:24:25.274 UTC] src/persistence/profile.cpp:290 : Debug: Loading tox save  "/home/diegoariel/.config/tox/dcapeletti.tox"
[12:24:25.274 UTC] src/core/core.cpp:131 : Debug: Core starting with IPv6 enabled
[12:24:25.584 UTC] src/widget/systemtrayicon.cpp:65 : Debug: Using GTK backend
[12:24:27.053 UTC] src/core/core.cpp:415 : Debug: "Connecting to 37.97.185.116:33445 (Yani)"
[12:24:27.054 UTC] src/core/core.cpp:415 : Debug: "Connecting to 163.172.136.118:33445 (LittleVulpix)"
[12:24:31.552 UTC] src/core/core.cpp:415 : Debug: "Connecting to v6.tox.zodiaclabs.org:33445 (stal)"
[12:24:37.176 UTC] src/core/core.cpp:415 : Debug: "Connecting to 128.199.199.197:33445 (wiiaam)"
[12:24:40.712 UTC] src/core/core.cpp:379 : Debug: Connected to the DHT
[12:24:40.712 UTC] src/persistence/profile.cpp:364 : Debug: Saving tox save to  "/home/diegoariel/.config/tox/dcapeletti.tox"
[12:24:52.255 UTC] src/widget/form/chatform.cpp:509 : Debug: enableCallButtons
[12:24:52.256 UTC] src/widget/form/chatform.cpp:522 : Debug: timer started!!
[12:24:55.159 UTC] src/audio/audio.cpp:386 : Debug: Opening audio output "Audio Interno Estéreo analógico"
[12:24:55.174 UTC] src/audio/audio.cpp:396 : Warning: Cannot open output audio device "Audio Interno Estéreo analógico"
[12:24:56.903 UTC] src/core/corefile.cpp:350 : Debug: Avatar transfer 0 to friend 0 accepted
[12:25:15.654 UTC] src/widget/form/chatform.cpp:509 : Debug: enableCallButtons
[12:25:15.657 UTC] src/widget/form/chatform.cpp:522 : Debug: timer started!!
[12:25:20.801 UTC] src/widget/form/chatform.cpp:475 : Debug: onCallTriggered
[12:25:20.805 UTC] src/core/coreav.cpp:232 : Debug: "Starting call with 0"
[12:25:20.805 UTC] src/audio/audio.cpp:345 : Debug: Opening audio input "Audio Interno Estéreo analógico"
[12:25:20.816 UTC] src/audio/audio.cpp:363 : Warning: Failed to initialize audio input device: "Audio Interno Estéreo analógico"
[12:25:20.816 UTC] src/audio/audio.cpp:293 : Warning: Failed to subscribe to audio input device.
[12:25:20.816 UTC] src/audio/audio.cpp:386 : Debug: Opening audio output "Audio Interno Estéreo analógico"
[12:25:20.819 UTC] src/audio/audio.cpp:400 : Debug: Opened audio output "Audio Interno Estéreo analógico"
[12:25:20.822 UTC] src/audio/audio.cpp:669 : Debug: Audio source 2 created. Sources active: 1
[12:25:20.822 UTC] src/audio/audio.cpp:345 : Debug: Opening audio input "Audio Interno Estéreo analógico"
[12:25:20.824 UTC] src/audio/audio.cpp:369 : Debug: Opened audio input "Audio Interno Estéreo analógico"
[12:25:20.825 UTC] src/audio/audio.cpp:298 : Debug: Subscribed to audio input device [ 1 subscriptions ]
[12:25:20.825 UTC] src/audio/audio.cpp:314 : Debug: Unsubscribed from audio input device [ 0 subscriptions left ]
[12:25:20.825 UTC] src/audio/audio.cpp:515 : Debug: Closing audio input
[12:25:26.008 UTC] src/widget/form/chatform.cpp:323 : Debug: onAvStart
AL lib: FreeContext: (0x7ff7bc009370) Deleting 1 Source(s)
[12:25:48.339 UTC] src/audio/audio.cpp:548 : Debug: Closing audio output
[12:25:48.339 UTC] src/audio/audio.cpp:386 : Debug: Opening audio output "Audio Interno Estéreo analógico"
[12:25:48.345 UTC] src/audio/audio.cpp:400 : Debug: Opened audio output "Audio Interno Estéreo analógico"
[12:25:48.355 UTC] src/audio/audio.cpp:345 : Debug: Opening audio input "Audio Interno Estéreo analógico"
[12:25:48.358 UTC] src/audio/audio.cpp:369 : Debug: Opened audio input "Audio Interno Estéreo analógico"
[12:25:48.363 UTC] src/audio/audio.cpp:669 : Debug: Audio source 2 created. Sources active: 1
[12:25:48.432 UTC] src/widget/form/settings/avform.cpp:339 : Debug: available Modes:
[12:25:48.432 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 30.000000, pixel format: yuyv422
[12:25:48.432 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 25.000000, pixel format: yuyv422
[12:25:48.432 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 20.000000, pixel format: yuyv422
[12:25:48.433 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 15.000000, pixel format: yuyv422
[12:25:48.433 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 10.000000, pixel format: yuyv422
[12:25:48.433 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 7.000000, pixel format: yuyv422
[12:25:48.433 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 5.000000, pixel format: yuyv422
[12:25:48.433 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 30.000000, pixel format: yuyv422
[12:25:48.433 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 25.000000, pixel format: yuyv422
[12:25:48.433 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 20.000000, pixel format: yuyv422
[12:25:48.433 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 15.000000, pixel format: yuyv422
[12:25:48.433 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 10.000000, pixel format: yuyv422
[12:25:48.433 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 7.000000, pixel format: yuyv422
[12:25:48.434 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 5.000000, pixel format: yuyv422
[12:25:48.434 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 30.000000, pixel format: yuyv422
[12:25:48.434 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 25.000000, pixel format: yuyv422
[12:25:48.434 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 20.000000, pixel format: yuyv422
[12:25:48.434 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 15.000000, pixel format: yuyv422
[12:25:48.434 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 10.000000, pixel format: yuyv422
[12:25:48.434 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 7.000000, pixel format: yuyv422
[12:25:48.434 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 5.000000, pixel format: yuyv422
[12:25:48.434 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 30.000000, pixel format: yuyv422
[12:25:48.434 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 25.000000, pixel format: yuyv422
[12:25:48.434 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 20.000000, pixel format: yuyv422
[12:25:48.434 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 15.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 10.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 7.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 5.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 30.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 25.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 20.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 15.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 10.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 7.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 5.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 720, FPS: 7.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 720, FPS: 5.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 800, FPS: 7.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 800, FPS: 5.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 960, FPS: 7.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 960, FPS: 5.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 1024, FPS: 7.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 1024, FPS: 5.000000, pixel format: yuyv422
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 30.000000, pixel format: mjpeg
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 25.000000, pixel format: mjpeg
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 20.000000, pixel format: mjpeg
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 15.000000, pixel format: mjpeg
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 10.000000, pixel format: mjpeg
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 7.000000, pixel format: mjpeg
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 640, height: 480, FPS: 5.000000, pixel format: mjpeg
[12:25:48.435 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 30.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 25.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 20.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 15.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 10.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 7.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 160, height: 120, FPS: 5.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 30.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 25.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 20.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 15.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 10.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 7.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 176, height: 144, FPS: 5.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 30.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 25.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 20.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 15.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 10.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 7.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 320, height: 240, FPS: 5.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 30.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 25.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 20.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 15.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 10.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 7.000000, pixel format: mjpeg
[12:25:48.436 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 352, height: 288, FPS: 5.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 720, FPS: 30.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 720, FPS: 25.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 720, FPS: 20.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 720, FPS: 15.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 720, FPS: 10.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 720, FPS: 7.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 720, FPS: 5.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 800, FPS: 30.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 800, FPS: 25.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 800, FPS: 20.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 800, FPS: 15.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 800, FPS: 10.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 800, FPS: 7.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 800, FPS: 5.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 960, FPS: 30.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 960, FPS: 25.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 960, FPS: 20.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 960, FPS: 15.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 960, FPS: 10.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 960, FPS: 7.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 960, FPS: 5.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 1024, FPS: 30.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 1024, FPS: 25.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 1024, FPS: 20.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 1024, FPS: 15.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 1024, FPS: 10.000000, pixel format: mjpeg
[12:25:48.437 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 1024, FPS: 7.000000, pixel format: mjpeg
[12:25:48.438 UTC] src/widget/form/settings/avform.cpp:201 : Debug: width: 1280, height: 1024, FPS: 5.000000, pixel format: mjpeg
[12:25:48.438 UTC] src/widget/form/settings/avform.cpp:353 : Debug: selected Modes:
[12:25:48.438 UTC] src/widget/form/settings/avform.cpp:271 : Debug: width: 640, height: 480, FPS: 30.000000, pixel format: mjpeg

[12:25:48.438 UTC] src/widget/form/settings/avform.cpp:271 : Debug: width: 320, height: 240, FPS: 30.000000, pixel format: mjpeg

[12:25:48.438 UTC] src/widget/form/settings/avform.cpp:271 : Debug: width: 160, height: 120, FPS: 30.000000, pixel format: mjpeg

[12:25:48.438 UTC] src/video/camerasource.cpp:375 : Debug: Closing device  "none"
[12:25:48.438 UTC] src/video/camerasource.cpp:289 : Debug: Opening device  "/dev/video0"
[12:25:49.200 UTC] src/audio/audio.cpp:298 : Debug: Subscribed to audio input device [ 1 subscriptions ]
Violación de segmento
```

**Comment by antis81** [comment]
Please provide some more information. This might be a duplicate of another issue, but I'm not sure what you're trying to say. You're talking about a crash or maybe the voice going silent? Btw. please attach log as file (this is now possible on Github) instead of inline text.

**Comment by sudden6** [comment]
@antis81 "Violación de segmento" probably means "Segmentation fault" thus a crash.

@dcapeletti It would be very helpful if you could translate error messages to English or temporarily switch the language to English, because not every developer speaks your native language.

**Comment by antis81** [comment]
@sudden6 Thanks, I didn't see that. Seems I need holiday. Translation not required: Log is already in English and it's the systems default language (not qTox), which produces this. If that is Windows, you're probably out of luck in that case, but IIRC Windows doesn't produce a SIGSEGV. :stuck_out_tongue:

@dcapeletti No further information required. But… I see you're on an older qTox version. Could you update? Because audio device initialization has changed and and probably fixes your problem.

**Comment by sudden6** [comment]
does this still happen with the newest qTox release?

**Comment by dcapeletti** [comment]
It's only been once, then again no longer happen. I can not find way to play it.

**Comment by ProMcTagonist** [comment]
The reverse is true for me. Occasionally qTox will only record my audio during an active call (according to pavucontrol) if I am inside the AV settings. Sometimes spamming Rescan Devices fixes it, sometimes not.

I will try to get logs if it happens again.

**Comment by Diadlo** [comment]
@ProMcTagonist seems your issue is #3538

**Comment by Diadlo** [comment]
Since issue was reproduce only once, and we have some AV changes, which may already fix the issue, I will close. It it will be reproduced again, please comment here or open new issue
